internal/plugins/nfs-shares.go

```diff
@@ -37,7 +37,7 @@ func (m *assetManagerNFS) parseAssetType(assetType db.AssetType) Option[assetTyp
 		return Some(assetTypeNFS{AllShares: true})
 	}
 
-	if nfsType, ok := strings.CutPrefix(string(assetType), "nfs-shares-type:"); ok {
+	if nfsType, ok := strings.CutPrefix(string(assetType), "nfs-shares-group:"); ok {
 		if shareTypeID, ok := m.shareTypeNameToID[nfsType]; ok {
 			return Some(assetTypeNFS{AllShares: false, ShareTypeName: nfsType, ShareTypeID: shareTypeID})
```

Comment:
```suggestion
	if nfsGroup, ok := strings.CutPrefix(string(assetType), "nfs-shares-group:"); ok {
		if shareGroupID, ok := m.shareTypeNameToID[nfsGroup]; ok {
			return Some(assetTypeNFS{AllShares: false, ShareTypeName: nfsGroup, shareGroupID: shareTypeID})
```
Don't we also need to rename those?